kivy/data/style.kv

```diff
@@ -235,8 +235,8 @@
             rgba: 1, 1, 1, int(not self.is_leaf)
         Rectangle:
             source: 'atlas://data/images/defaulttheme/tree_%s' % ('opened' if self.is_open else 'closed')
-            size: self.height / 2.0, self.height / 2.0
```

Comment:
Size of arrow gets big if the row gets big in mobile (we must use big rows on mobile)